compiler/damlc/daml-doc/src/DA/Daml/Doc/HaddockParse.hs

```diff
@@ -269,6 +269,9 @@ getClsDocs ctx@DocCtx{..} (DeclData (L _ (TyClD _ c@ClassDecl{..})) docs) = do
     subdocs = memberDocs c
 getClsDocs _ _ = Nothing
 
+unknownType :: DDoc.Type
```

Comment:
Do we have an example where this is used?

Reply:
Not really. It's used in the `adt2rst` function to make it more robust against missing type information. AFAIK this happens only with `Bool` and `[]` in `GHC.Types` right now. The former doesn't actually use `_`. The latter does use it for the cons constructor, but it's something to fix later or never.

Reply:
So, I guess the answer isn't "Not really" but "only in cases where the type information is missing because of weirdness with primitives".